server/config.go

```diff
@@ -772,6 +777,18 @@ func ParseUrls(s string) ([]url.URL, error) {
 	return urls, nil
 }
 
+// SetupLogger setup the logger.
+func (c *Config) SetupLogger() error {
+	lg, syncer, err := log.InitLogger(&c.Log)
+	if err != nil {
+		return err
+	}
+	c.logger = lg
+	c.logSyncer = syncer
+	log.ReplaceGlobals(lg)
```

Comment:
It looks a bit strange here. I think `InitLogger` and `ReplaceGlobal` are more suitable in the main function.